packages/env-utils/src/envUtils.ts

```diff
@@ -112,7 +114,7 @@ const isLinux = () => {
     return getPlatform().toLowerCase().startsWith('linux');
 };
 
-const isCodesignBuild = () => process.env.IS_CODESIGN_BUILD === 'true';
```

Comment:
This is not correct. For production web, it would be replaced during build-time using `webpack.DefinePlugin` for something like:

```js
const isCodesignBuild = () => (hasProcess ? "true" === 'true' : false);
```
But `hasProcess` will be falsy on the web, so `isCodesignBuild` will always return `false` on the web.

Reply:
Ok, thanks for letting me know.  I actually had no idea it works this way. In that case, it only makes sense for `isWeb` (which is the only one I need right now), correct? Or do you have any proposed solution for how to address this?

Reply:
Done 🙏 Sorry, I had to rebase to remove the commit.